### DISCUSSION TITLE
Provide units for return values of canister status

### DISCUSSION THREAD

**Discussion by domwoe** [dfinity/sdk#2800] [Feature Requests]
When running `dfx canister status` currently only the balance is returned with a unit (cycles). It would be helpful to return all values with units. This would be helpful in particular for the freezing threshold which confuses people a lot.

Furthermore, it would be helpful if we could return the freezing threshold not only in seconds but also in cycles.

**Comment by vincent-dfinity** [comment]
Done at https://github.com/dfinity/sdk/pull/4064.
